### ISSUE TITLE
LV2 category missing

### ISSUE THREAD

**Issue by falkTX** [davemollen/dm-GrainDelay#1]
Hi there!
This delay is quite awesome, so good that we hope to be able to have it on the MOD plugin store soon.

Being based on rust was the first challenge, this is the first plugin going to the store that is based on rust.
Today I was able to get things going just enough to enable rust based plugins to build.
You can see the current mod-plugin-builder project at https://github.com/moddevices/mod-plugin-builder/blob/master/plugins/package/dm-graindelay/dm-graindelay.mk
I will some cloud builder changes so that rust tools are available by default there.

With that going, there are a few things that need attention, I opened this first ticket for the first one. And also as introduction.

Basically, the LV2 plugin category should be set as Delay.
I can send some pull requests for these small details, alternatively I can write a small list if you prefer to do them yourself.

Thanks again

**Comment by davemollen** [comment]
Hi Filipe, that's awesome! Feel free to send pull requests. I'd also be more than happy to look into feature/fix requests. Let's see it as a collaboration.

I'm curious about the build pipeline. Very nice that you're including rust. Because the whole docker setup I made isn't that neat.

I might have some questions later aswell. So hopefully we can keep in touch.

Nice to hear from you.

Dave

**Comment by falkTX** [comment]
Issues were fixed, thank you!
Plugin is now on the beta store, with a preview/description as usual on the pedalboards site too. URL being https://pedalboards.moddevices.com/plugins/aHR0cHM6Ly9naXRodWIuY29tL2RhdmVtb2xsZW4vZG0tR3JhaW5EZWxheQ==

Are you already on the MOD forums?
If so, would be cool if you would post about your own plugin in the "beta plugins" category, https://forum.mod.audio/c/plugins/plugins-beta-store/35
Jesse (from MOD team) has been using it quite a lot, we would like push the plugin to stable soon but having it for a bit in beta to allow more widespread testing.
You are ok with this, right? Or do you still want to work on the plugin a bit more?

**Comment by davemollen** [comment]
I'm on the forum, yes. I'll post something soon.

And I don't need to do any work now. Let's see how the beta phase goes!
